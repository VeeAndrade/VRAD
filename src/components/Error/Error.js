import React from 'react'

const Error = (errorMessage) => {
  return (
    <section>
      <h1>There was an error: {errorMessage}</h1>
    </section>
  )
}

export default Error
