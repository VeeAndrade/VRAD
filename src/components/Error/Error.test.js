import React from 'react';
import  { shallow } from 'enzyme';
import Error from './Error';

describe('Error', () => {
  it('should match the snapshot with all the data passed in correctly', () => {

    const wrapper = shallow(<Error
      errorMessage='There was an error retrieving your data !'
    />)
    expect(wrapper).toMatchSnapshot();
  })
})
