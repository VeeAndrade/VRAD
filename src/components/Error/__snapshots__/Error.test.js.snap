// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should match the snapshot with all the data passed in correctly 1`] = `
<section>
  <h1>
    There was an error: 
    <Component />
  </h1>
</section>
`;
